guard/src/commands/validate.rs

```diff
@@ -279,37 +341,23 @@ or rules files.
             None => None,
         };
 
-        let verbose = app.is_present(VERBOSE.0);
+        let verbose = app.get_flag(VERBOSE.0);
 
-        let data_type = match app.value_of(TYPE.0) {
-            Some(t) => {
-                if t == "CFNTemplate" {
-                    CFNTemplate
-                } else {
-                    Type::Generic
-                }
-            }
+        let data_type = match app.get_one::<String>(TYPE.0) {
+            Some(t) => Type::from(t.as_str()),
             None => Type::Generic,
         };
 
-        let output_type = match app.value_of(OUTPUT_FORMAT.0) {
-            Some(o) => {
-                if o == "single-line-summary" {
-                    OutputFormatType::SingleLineSummary
-                } else if o == "json" {
-                    OutputFormatType::JSON
-                } else {
-                    OutputFormatType::YAML
-                }
-            }
+        let output_type = match app.get_one::<String>(OUTPUT_FORMAT.0) {
+            Some(o) => OutputFormatType::from(o.as_str()),
             None => OutputFormatType::SingleLineSummary,
         };
 
         let summary_type: BitFlags<SummaryType> =
-            app.values_of(SHOW_SUMMARY.0)
+            app.get_many::<String>(SHOW_SUMMARY.0)
```

Comment:
`SHOW_SUMMARY` does not have multiple args, right? `get_many` is only supposed to be used when we are using the same option multiple times to collect all values, correct? Like for `RULES` and `DATA` it makes sense.

We should use `get_one()` here no?

Reply:
Correct me if Im wrong but this was the previous show-summary definition 

`.arg(Arg::with_name(SHOW_SUMMARY.0).long(SHOW_SUMMARY.0).short(SHOW_SUMMARY.1).takes_value(true).use_delimiter(true).multiple(true)
                .possible_values(["none", "all", "pass", "fail", "skip"])`
                
this multiple(true) implies that its a vector of String values.